client/components/Run/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Scenario from '@components/Scenario';
+
+class Run extends Component {
```

Comment:
This is looking good, but I think it might be more set up than we need. Initially I had pictured viewing a moment and starting a run to be the same thing. Perhaps we don't need a whole Run component and can instead add the run specific to the Scenario component and only execute it if `isScenarioRun` is true? What do you think of that approach?